src/cos_integration.py

```diff
@@ -100,45 +100,25 @@ def get_metrics_endpoints(self, node_name: str, token: str, cluster_name: str) -
             "replacement": node_name,
         }
 
+        components = [
+            ("kube-proxy", "/metrics", "http", "localhost:10249"),
+            ("apiserver", "/metrics", "https", "localhost:6443"),
+            ("kube-scheduler", "/metrics", "https", "localhost:10259"),
+            ("kube-controller-manager", "/metrics", "https", "localhost:10257"),
+        ]
```

Comment:
Should we have this include another key like `instance-relabel : []` or instance-relabel :[instance_relabel]`?